spec/classes/repos_spec.rb

```diff
@@ -2,6 +2,72 @@
 
 require 'spec_helper'
 describe 'kubernetes::repos', type: :class do
+  context 'with Debian and default params' do
+    let(:facts) do
+      {
+        osfamily: 'Debian', # needed to run dependent tests from fixtures puppetlabs-apt
+        kernel: 'Linux',
+        os: {
+          family: 'Debian',
+          name: 'Ubuntu',
+          release: {
+            full: '16.04'
+          },
+          distro: {
+            codename: 'xenial'
```

Comment:
This has nothing to do with newer distributions. Google was releasing all packages under `xenial` codename because the binary packages weren't dependent any specific distribution.